src/sections/bannerBlock/BannerBlock.module.scss

```
@@ -4,43 +4,34 @@
 	background-image: url('../../assets/bannerBlock/background.png');
 	background-size: cover;
 	background-repeat: no-repeat;
-	height: 885px;
+	height: 900px;
 	width: 100%;
 	display: flex;
 	position: relative;
 }
 
-.bgImages {
-	height: 100%;
-	position: absolute;
-	display: flex;
+.leftBlock {
+	background-image: url('../../assets/bannerBlock/leftsSde.png');
 	width: 100%;
+	background-size: 70% 100%;
+	background-repeat: no-repeat;
```

Don't forget about the `background-position` property. It can help you.
```suggestion
	background-size: 70% 100%;
	background-repeat: no-repeat;
	background-position: bottom;
```